File: prime.c
#include <stdio.h>
int prime(int n)
{
    int counter=0;
    for(int i=1;i<(n+1);++i)
    {
        if(n%i==0)
            ++counter;
    }
    if (counter==2)
        return 1;
    else 
        return 0;
}
void main()
{
    int n;
    scanf("%d", &n);
    if (prime(n)==1)
        printf("Prime number");
    else    
        printf("Not a Prime number");
}

File: prime.py
import math
def prime(n):
    counter=1
    number=n
    for i in range(2,int(math.sqrt(n)+1)):
        if (number%i==0): counter=0
    if (counter==1):
        print ("prime number")
    else:
        print ("Not prime number")
        
n=int(input());
prime(n)